### ISSUE TITLE
[deeplab] Train DeepLabV3+ with NYU-Depth V2 dataset

### ISSUE THREAD

**Issue by cclo-astri** [tensorflow/models#8602]
Hi all,

I just tried to use NYU-Depth V2 dataset to train DeepLabV3+. But the loss value does not converge to a lower value. Here is the training settings:

- Input images manually shrunk to a resolution with 512x512
- Train images: 796, eval images: 655
- 14 classes (13 classes, id 1-13 + 1 background, id = 0)
- ignore_label = 0

1. Using ADE20K base model, final loss is around 0.4 - 0.5, overall mIoU: 0.6609779
    - train_crop_size 256x256, bs=8, last_layers_contain_logits_only = true, initialize_last_layer = false
    - eval_crop_size 513x513

1. Using ADE20K base model, final loss is around 0.6, overall mIoU: 0.620792508
    - train_crop_size 256x256, bs=8, last_layers_contain_logits_only = false, initialize_last_layer = false
    - eval_crop_size 513x513

1. Using PASCAL VOC base model, final loss is around 0.8, overall mIoU: 0.525445759
    - train_crop_size 256x256, bs=8, last_layers_contain_logits_only = true, initialize_last_layer = false
    - eval_crop_size 513x513

1. Using PASCAL VOC base model, final loss is around 0.8, overall mIoU: 0.551259339
    - train_crop_size 256x256, bs=8, last_layers_contain_logits_only = false, initialize_last_layer = false
    - eval_crop_size 513x513

Are these results acceptable ? 

If I want to increase the mIoU value for the training with NYU-Depth V2 dataset, what parameters should I change and test ? 
- Learning rate ?
- Momentum ?
- Weights ?

And as I am a newbie in deep learning field, I do not know the exact usage of above parameters and where I could change the "weight" of the model.

Please help. 

Thanks.
